README.md

```diff
@@ -201,25 +214,30 @@ An example of such a mask would be `3XX, 4XX, 5XX`. In this case, all 300s, 400s
 
 ## Table API Connector Options
 ### HTTP TableLookup Source
-| Option       | Required | Description/Value                                                                                        |
-|--------------|----------|----------------------------------------------------------------------------------------------------------|
-| connector    | required | The Value should be set to _rest-lookup_                                                                 |
-| format       | required | Flink's format name that should be used to decode REST response, Use `json` for a typical REST endpoint. |
-| url          | required | The base URL that should be use for GET requests. For example _http://localhost:8080/client_             |
-| asyncPolling | optional | true/false - determines whether Async Pooling should be used. Mechanism is based on Flink's Async I/O.   |
+| Option                                       | Required | Description/Value                                                                                                             |
+|----------------------------------------------|----------|-------------------------------------------------------------------------------------------------------------------------------|
+| connector                                    | required | The Value should be set to _rest-lookup_                                                                                      |
+| format                                       | required | Flink's format name that should be used to decode REST response, Use `json` for a typical REST endpoint.                      |
+| url                                          | required | The base URL that should be use for GET requests. For example _http://localhost:8080/client_                                  |
+| asyncPolling                                 | optional | true/false - determines whether Async Pooling should be used. Mechanism is based on Flink's Async I/O.                        |
+| gid.connector.http.lookup.error.code         | optional | List of HTTP status codes that should be treated as errors by HTTP Source, separated with comma.                              |
+| gid.connector.http.lookup.error.code.exclude | optional | List of HTTP status codes that should be excluded from the `gid.connector.http.lookup.error.code` list, separated with comma. |
```

Comment:
sorry what was changed here?

Reply:
I added `gid.connector.http.lookup.error.code` and `gid.connector.http.lookup.error.code.exclude` to the list of parameters. And reformatted the whole table using IntelliJ :)

Reply:
right that reformat confused me :)